zztboard.js

```diff
@@ -103,25 +103,13 @@ ZZTBoard.prototype.remove = function(x, y)
 
 ZZTBoard.load = function(boardID, x, y)
```

Comment:
Probably should make this function more generic since walking off the board is another way of moving to a different board.